### PR TITLE
Refactor bounty card component

### PR DESCRIPTION
**Pull request by corwinharrell**
- Refine bounty card styles and styles of components within bounty card:
circle, payout, text, and variables.

screen shot 2018-06-10 at 6 43 18 pm

### CODE REVIEW

**Comment by corwinharrell** [comment]
@somethiiing @villanuevawill if you guys could double-check any changes I make to .js files just to make sure I don't screw anything up 👍

**Comment by corwinharrell** [comment]
Also, @somethiiing we should create a separate "Avatar" component. "Circles" is too ambiguous and an avatar and a metric encased in a circle don't really serve similar functions. Their only similarity is their shape, and I don't think that justifies structuring avatars and metrics as props of the same component.

**Comment by corwinharrell** [comment]
A couple other thoughts on bounty card:

- We'll need to concatenate the user ID string 
- The markup structure of the middle card column seems to be causing some odd alignment issues text slightly offset) @somethiiing can we simplify how this is written? Seems to be a bit of div-itis going on here.

**Comment by villanuevawill** [comment]
@somethiiing agree on the avatar vs. circle thing.

File: src/components/BountyCard/BountyCard.module.scss [hunk 1]
```diff
@@ -1,50 +1,57 @@
 @import "~styles/variables.scss";
 
 .bountyCardContainer {
+  border-left: 3px solid transparent;
```
Comment (villanuevawill) [inline src/components/BountyCard/BountyCard.module.scss]: should these also scale with screen size?

Comment (villanuevawill) [inline src/components/BountyCard/BountyCard.module.scss]: Should these also scale with screen size?

File: src/components/BountyCard/index.js [hunk 2]
```diff
@@ -54,31 +60,43 @@ const BountyCard = props => {
...
         <div className={`${styles.dataCell}`}>
           <div className={`${styles.dataIcon}`}>
-            <Text color="grey">
+            <Text color="grey" style="Body">
               <FontAwesomeIcon icon={faClock} />
             </Text>
           </div>
-          <Text color="black">{moment(deadline).fromNow(true)}</Text>
+          <Text color="black" style="Body" weight="font-weight-bold">
+            {moment(deadline).fromNow(true)}
```
Comment (villanuevawill) [inline src/components/BountyCard/index.js]: @corwinharrell have you looked at/played with all moment's formatters? We can always change the format, etc if need be.

**Comment by villanuevawill** [comment]
@corwinharrell why is your yarn.lock different? I don't see any packages added or removed.

**Comment by villanuevawill** [comment]
*the request review is related to the yarn.lock file. The rest looks fine, but am curious about the few variables that should probably be responseive with screensize?

File: src/components/BountyCard/BountyCard.module.scss (hunk 1 above)
Comment (corwinharrell) [reply src/components/BountyCard/BountyCard.module.scss]: No this should remain static. This and things like border-radii on containers are things we don't want fluctuating.

**Comment by corwinharrell** [comment]
@villanuevawill I have no idea why the yarn.lock file is different. I didn't touch it. Do I need to do anything about that?

File: src/components/BountyCard/index.js (hunk 2 above)
Comment (corwinharrell) [reply src/components/BountyCard/index.js]: @villanuevawill I have not but that's cool.

**Pull request merged**